index.html

```diff
@@ -11,7 +11,8 @@
     <main>
       <canvas id="board"></canvas>
       <!--This button is just for see the functionality, so use it or delete ;)-->
-      <button id="show">SHOW</button>
+      <button id="reset">RESET</button>
+      <button id="start">START</button>
```

Comment:
When you change the text inside (start->stop) the size of the button change, so I think it is not correct, so, we can add somethin in css to make the buttons fixed.